internal/tofu/transform_provider.go

```diff
@@ -227,26 +237,20 @@ func (t *ProviderFunctionTransformer) Transform(g *Graph) error {
 		return nil
 	}
 
-	// Locate all providers in the graph
-	providers := providerVertexMap(g)
-
-	type providerReference struct {
-		path  string
-		name  string
-		alias string
-	}
+	// Locate all providerVerts in the graph
+	providerVerts := providerVertexMap(g)
 	// LuT of provider reference -> provider vertex
```

Comment:
i think this comment needs updating now, as we're not looking up provider vertexes anymore here, instead it's the function refs